src/server/system_services/pool_controllers.js

```diff
@@ -159,21 +110,16 @@ class ManagedStatefulSetPoolController extends PoolController {
 // have no control over the underlying storage
 // ----------------------------------------------
 class UnmanagedStatefulSetPoolController extends PoolController {
-    constructor(system_name, pool_name) {
-        super(system_name, pool_name);
-        this._statefulset_name = _get_statefulset_name(system_name, pool_name);
-    }
-
     // Just return the configuration to apply in order to create
     // the statefulset.
     async create(agent_count, agent_install_conf, agent_profile) {
         const pool_k8s_conf = await _get_k8s_conf({
-            statefulset_name: this._statefulset_name,
+            pool_name: this.pool_name,
             agent_count,
             agent_install_conf,
             ...agent_profile
         });
-        return yaml_utils.stringify(pool_k8s_conf);
+        return yaml.stringify(pool_k8s_conf, 6, 2);
```

Comment:
`yaml_utils.stringify()` handle special cases that `yaml.stringify` do not. 
Is there a reason to not using it?

Reply:
it just didn't work in my case - I guess it handled specifically issues in the old statefulset format the was just not needed anymore. But anyway this review starting to be unrelevant as I'm changing the backingstore format to using secrets, so it will change again anyway